src/test/java/nextstep/subway/unit/SubwayMapTest.java

```diff
@@ -34,38 +31,64 @@ void setUp() {
         이호선 = new Line("2호선", "red");
         삼호선 = new Line("3호선", "red");
 
-        신분당선.addSection(강남역, 양재역, 3);
-        이호선.addSection(교대역, 강남역, 3);
-        삼호선.addSection(교대역, 남부터미널역, 5);
-        삼호선.addSection(남부터미널역, 양재역, 5);
+        신분당선.addSection(강남역, 양재역, 3, 5);
+        이호선.addSection(교대역, 강남역, 3, 6);
+        삼호선.addSection(교대역, 남부터미널역, 5, 1);
+        삼호선.addSection(남부터미널역, 양재역, 5, 2);
     }
 
     @Test
-    void findPath() {
+    void findPathByDistance() {
         // given
         List<Line> lines = Lists.newArrayList(신분당선, 이호선, 삼호선);
         SubwayMap subwayMap = new SubwayMap(lines);
 
         // when
-        Path path = subwayMap.findPath(교대역, 양재역);
+        Path path = subwayMap.findPath(교대역, 양재역, PathType.valueOf("DISTANCE"));
```

Comment:
테스트에서는 PathType을 명시해줘도 좋을 것 같아요 😃 
```suggestion
        Path path = subwayMap.findPath(교대역, 양재역, PathType.DISTANCE);
```